src/chainparams.h

```diff
@@ -16,6 +16,7 @@
 
 struct CDNSSeedData {
     std::string host;
+    //! Implies that the dnsseed can filter for at least xGetDesireableSerivceFlags(NODE_NONE)
```

Comment:
Typo: **x**GetDesir**e**ableServiceFlags?

Reply:
x was deliberate (as thats the dns prefix), the e is just because i cant spell desirable.